charts/sequencer/templates/statefulsets.yaml

```diff
@@ -16,6 +16,11 @@ spec:
       labels:
         app: {{ .Values.moniker }}-sequencer
     spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 3000
+        fsGroup: 2000
+        supplementalGroups: [4000]
```

Comment:
@WafflesVonMaple feel a bit out of my depth on these settings. I'm also certain there is more configuration going on here than necessary, but these setting work.

Reply:
runAsUser: # hardcode user to non-root if not set in dockerfile
runAsGroup: # hardcode group to non root if not set in Dockerfile
runAsNonRoot: # hardcode to non-root. Redundant if dockerfile is set to USER 1000

General safety dictates runAsUser/Group are not explicit replacement for runAsNonRoot as the ID must exist in the container for it to run. 

runAsNonRoot also limits access to host resources that might mistakenly get exposed to the container 

fsGroup/GroupChangePolicy defines group which kube will change perm of all files in volumes mounted. Caution to be exercised as changing group ownership of an entire volume can cause startup delay or other issues